scalability_and_performance/recommended-cluster-scaling-practices.adoc

```diff
@@ -10,4 +10,13 @@ your {product-title} cluster. You scale the worker machines by increasing or
 decreasing the number of replicas that are defined in the worker MachineSet.
 
 include::modules/recommended-scale-practices.adoc[leveloffset=+1]
+
 include::modules/machineset-modifying.adoc[leveloffset=+1]
+
+include::modules/machine-health-checks-about.adoc[leveloffset=+1]
+
+include::modules/machine-health-checks-resource.adoc[leveloffset=+1]
+
+include::modules/machine-health-checks-creating.adoc[leveloffset=+1]
+
+>>>>>>> Add recommendation to enable machine health checks
```

Comment:
@chaitanyaenr Looks like we now have some errant text in >>>>>, probably leftover from a rebase. Once that's addressed, we should be all set to merge. Thanks!

Reply:
@ahardin-rh Fixed, thanks.